Mathlib/Algebra/Category/AlgebraCat/Limits.lean

```diff
@@ -105,16 +105,30 @@ def limitConeIsLimit (F : J ⥤ AlgebraCatMax.{v, w} R) : IsLimit (limitCone.{v,
     apply Subtype.ext
     ext j
     simp [forget_map_eq_coe, AlgHom.map_one, Functor.mapCone_π_app]
+    -- This used to be `rw`, but we need `erw` after leanprover/lean4#2644
+    erw [map_one]
```

Comment:
There wasn't a `rw` in the original code?

Reply:
I started cleaning up things but about to fall into a black hole for a few days.